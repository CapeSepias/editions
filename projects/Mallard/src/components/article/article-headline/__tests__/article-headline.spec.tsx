import React from 'react'
import TestRenderer, { ReactTestRendererJSON } from 'react-test-renderer'
import { ArticleHeadline } from '..'

describe('ArticleHeadline', () => {
    it('should show a news headling', () => {
        const component: ReactTestRendererJSON | null = TestRenderer.create(
            <ArticleHeadline type="news">
                England’s Suzy Petty hopes to help GB hockey team make it to
                Olympics
            </ArticleHeadline>,
        ).toJSON()
        expect(component).toMatchSnapshot()
    })
    it('should show a long read headling', () => {
        const component: ReactTestRendererJSON | null = TestRenderer.create(
            <ArticleHeadline type="longRead">
                England’s Suzy Petty hopes to help GB hockey team make it to
                Olympics
            </ArticleHeadline>,
        ).toJSON()
        expect(component).toMatchSnapshot()
    })
})
