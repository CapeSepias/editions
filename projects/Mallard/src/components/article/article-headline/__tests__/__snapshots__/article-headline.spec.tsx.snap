// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleHeadline should show a long read headling 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#121212",
        "flexShrink": 0,
        "fontFamily": "GHGuardianHeadline-Regular",
        "fontSize": 28,
        "lineHeight": 30,
      },
      Object {
        "marginRight": 24,
        "marginTop": 3,
      },
    ]
  }
>
  England’s Suzy Petty hopes to help GB hockey team make it to Olympics
</Text>
`;

exports[`ArticleHeadline should show a news headling 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#121212",
        "flexShrink": 0,
        "fontFamily": "GHGuardianHeadline-Regular",
        "fontSize": 28,
        "lineHeight": 30,
      },
      Object {
        "marginRight": 24,
        "marginTop": 3,
      },
    ]
  }
>
  England’s Suzy Petty hopes to help GB hockey team make it to Olympics
</Text>
`;
